app/views/api/v1/users/show.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<section class="user-info">
+  <h2>Hello, <%= current_user.username %></h2>
+  <ul>
+    <li>Your current zipcode: <%= current_user.zipcode %></li>
+  </ul>
+</section>
+
+<section class="favorite-artists">
+  <h2>Favroties:</h2>
+  <ul>
+    <% if @favorites == [] %>
+      <li>No Favorite Artists... yet</li>
+    <% else %>
+      <% @favorites.each do |favorite| %>
+      <div class= <%= "artist-#{favorite.artist_id}"%> >
+        <% name = Artist.find(favorite.artist_id).name %>
```

Comment:
Need to put this logic in controller or model helper method

Reply:
Done. Thanks!